Neos.Flow/Tests/Unit/Mvc/Controller/ActionControllerTest.php

```diff
@@ -201,45 +201,14 @@ public function processRequestThrowsExceptionIfRequestedActionIsNotPublic()
     /**
      * @test
      */
-    public function processRequestInjectsControllerContextToView()
-    {
-        $this->actionController = $this->getAccessibleMock(ActionController::class, ['resolveActionMethodName', 'initializeActionMethodArguments', 'initializeActionMethodValidators', 'resolveView', 'callActionMethod', 'initializeController']);
-        $this->actionController->method('resolveActionMethodName')->willReturn('indexAction');
-
-        $this->inject($this->actionController, 'objectManager', $this->mockObjectManager);
-        $this->inject($this->actionController, 'controllerContext', $this->mockControllerContext);
-        $this->inject($this->actionController, 'request', $this->mockRequest);
-
-        $this->inject($this->actionController, 'arguments', new Arguments([]));
-
-        $mockMvcPropertyMappingConfigurationService = $this->createMock(Mvc\Controller\MvcPropertyMappingConfigurationService::class);
-        $this->inject($this->actionController, 'mvcPropertyMappingConfigurationService', $mockMvcPropertyMappingConfigurationService);
-
-        $mockHttpRequest = $this->getMockBuilder(ServerRequestInterface::class)->disableOriginalConstructor()->getMock();
-        $this->mockRequest->expects(self::any())->method('getHttpRequest')->will(self::returnValue($mockHttpRequest));
-
-        $mockResponse = new Mvc\ActionResponse;
-        $mockResponse->setContentType('text/plain');
-        $this->inject($this->actionController, 'response', $mockResponse);
-
-        $mockView = $this->createMock(Mvc\View\ViewInterface::class);
-        $mockView->expects(self::once())->method('setControllerContext')->with($this->mockControllerContext);
-        $this->actionController->expects(self::once())->method('resolveView')->will(self::returnValue($mockView));
-        $this->actionController->expects(self::once())->method('resolveActionMethodName')->will(self::returnValue('someAction'));
-
-        $this->actionController->processRequest($this->mockRequest, $mockResponse);
-    }
-
-    /**
-     * @test
-     */
-    public function processRequestInjectsSettingsToView()
+    public function processRequestInjectsSettingsAndRequestToView()
     {
         $this->actionController = $this->getAccessibleMock(ActionController::class, ['resolveActionMethodName', 'initializeActionMethodArguments', 'initializeActionMethodValidators', 'resolveView', 'callActionMethod']);
         $this->actionController->method('resolveActionMethodName')->willReturn('indexAction');
 
         $this->inject($this->actionController, 'objectManager', $this->mockObjectManager);
         $this->inject($this->actionController, 'controllerContext', $this->mockControllerContext);
+        $this->inject($this->actionController, 'request', $this->mockRequest);
```

Comment:
One last question, do we want to use `request` as naming here or preserve that slot for a pretty psr request, despite that in Fusion request is an ActionRequest (which will not change anytime?)

Do we all agree to keep the ActionRequest for some time and have it hold all the context, like matched controller, base uri and routing parameters?